index.bs

```diff
@@ -568,6 +571,18 @@ through a process called [=sensor fusion=].
 [=calibration|Calibrated=] [=raw sensor readings=] are referred to as <dfn>sensor readings</dfn>,
 whether or not they have undergone [=sensor fusion=].
 
+A <dfn id="concept-platform-sensor">platform sensor</dfn> is a term for a software entity which
```

Comment:
The term platform sensor refers to an implementation, with which the user agent interfaces with, that provides access to sensor readings

Reply:
done